t/ocsp.t

```diff
@@ -16,6 +16,14 @@ no_long_string();
 $ENV{TEST_NGINX_LUA_PACKAGE_PATH} = "$t::TestCore::lua_package_path";
 $ENV{TEST_NGINX_HTML_DIR} ||= html_dir();
 
+$ENV{TEST_NGINX_SERVER_SSL_PORT} ||= 4443;
+
+add_block_preprocessor(sub {
+    if ($Test::Nginx::Util::Randomize) {
+        $ENV{TEST_NGINX_SERVER_SSL_PORT} = gen_random_port;
```

Comment:
It's easier to add automatic support for special variables `$TEST_NGINX_RANDOM_PORT1`, `$TEST_NGINX_RANDOM_PORT2`, and etc., in the Test::Nginx::Socket test scaffold. So that the test files can directly use and reference those special variables without such duplicate boilerplate code in every related .t file.